hnn_core/cell.py

```diff
@@ -341,6 +363,39 @@ def gid(self, gid):
         else:
             raise RuntimeError('Global ID for this cell already assigned!')
 
+    def distance_section(self, target_sec_name, base_sec_name, base_sec_end):
```

Comment:
```suggestion
    def distance_section(self, target_sec_name, base_node):
```

to be consistent with the tree representation? seems to simplify lot of stuff below if you stick to well-defined data structure